chromium_src/third_party/blink/renderer/modules/storage/storage_namespace.h

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_MODULES_STORAGE_STORAGE_NAMESPACE_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_MODULES_STORAGE_STORAGE_NAMESPACE_H_
+
+#define BRAVE_STORAGE_NAMESPACE_H \
+  const String& namespace_id() { return namespace_id_; }
```

Comment:
as mentioned in a previous version of this PR this patch is not necessary. You can get the namespace id from the WebViewClient doing something like this
```
auto* web_frame = WebLocalFrameImpl::FromFrame(document->GetFrame())
WebViewImpl* webview = web_frame->ViewImpl();
if (webview && !webview->Client())
  webview->Client()->GetSessionStorageNamespaceId()    
```

Reply:
Okay. I've pushed a new version of this branch that eliminates the patch.